### ISSUE TITLE
0.6 master server register issues

### ISSUE THREAD

**Issue by ChillerDragon** [ddnet/ddnet#8722]
There were a few reports in #developer about master server registration issues. I also noticed it on my servers now. It seems like the 0.6 protocol fails to register. Because the 0.7 still works I would blame https://github.com/ddnet/ddnet/pull/5949.

```
2024-08-14 02:17:03 I server: git revision hash: 5a2891933cb91257
2024-08-14 02:17:03 I server: +-------------------------+
2024-08-14 02:17:03 I server: | rcon password: 'eK3P2V' |
2024-08-14 02:17:03 I server: +-------------------------+
2024-08-14 02:17:04 I register/6/ipv6: registering...
2024-08-14 02:17:04 I register/6/ipv4: registering...
2024-08-14 02:17:04 I register/7/ipv6: registering...
2024-08-14 02:17:04 I register/7/ipv4: registering...
2024-08-14 02:17:04 I register/7/ipv6: registering...
2024-08-14 02:17:05 I register/6/ipv6: registering...
2024-08-14 02:17:05 I register/7/ipv4: registering...
2024-08-14 02:17:19 I register/6/ipv4: registering...
2024-08-14 02:17:19 E register/6/ipv4: ERROR: the master server reports that clients can not connect to this server.
2024-08-14 02:17:19 E register/6/ipv4: ERROR: configure your firewall/nat to let through udp on port 8303.
2024-08-14 02:17:35 I register/6/ipv4: registering...
2024-08-14 02:17:35 E register/6/ipv4: ERROR: the master server reports that clients can not connect to this server.
2024-08-14 02:17:35 E register/6/ipv4: ERROR: configure your firewall/nat to let through udp on port 8303.
2024-08-14 02:17:50 I register/6/ipv4: registering...
2024-08-14 02:17:50 E register/6/ipv4: ERROR: the master server reports that clients can not connect to this server.
2024-08-14 02:17:50 E register/6/ipv4: ERROR: configure your firewall/nat to let through udp on port 8303.
2024-08-14 02:18:05 I register/6/ipv4: registering...
```

**Comment by ChillerDragon** [comment]
Yes this looks like I broke something on the protocol level for connless packets.
Left is the master branch sending weird stuff. Right side is a old ddnet server.

![image](https://github.com/user-attachments/assets/2746f9dd-4192-4dee-bd50-6967cdb8c051)

**Comment by heinrich5991** [comment]
@ChillerDragon Please put the fixed issue into the commit/PR body, not the title. GitHub doesn't use "fixes #<number>" from the title.

Fixed by #8725.
